docs/mkdocs.yml

```
@@ -8,14 +8,19 @@ edit_uri: https://github.com/hedgedoc/hedgedoc/edit/master/docs/content/
 nav:
   - Home: index.md
   - Installation:
-      - 'Manual Installation': setup/manual-setup.md
-      - 'Reverse Proxy': setup/reverse-proxy.md
-      - Docker: setup/docker.md
-      - Cloudron: setup/cloudron.md
-      - Heroku: setup/heroku.md
-      - LinuxServer: setup/docker-linuxserver.md
-      - Yunohost: setup/yunohost.md
+      - Official installation methods:
+          - Docker: setup/docker.md
+          - 'Manual Installation': setup/manual-setup.md
+          - Heroku: setup/heroku.md
+      - Community installation methods:
+          - Cloudron: setup/cloudron.md
+          - LinuxServer: setup/docker-linuxserver.md
```

```suggestion
          - 'LinuxServer Docker': setup/docker-linuxserver.md
```
This sounds a bit weird, so we might add 'Image' both here and for our own image

---

Done